### ISSUE TITLE
About pointpillar training

### ISSUE THREAD

**Issue by GuoliangPeng** [w111liang222/lidar-slam-detection#4]
Hello, do you have a training framework for open-source lidar detection and how to use my own data for training？ And what is the accuracy of the improved pointpillar？

**Comment by w111liang222** [comment]
Hi, 
We use the OpenPCDet framework to train the models in LSD, it supports to use custom dataset.

We compared the LSD's lidar detection model(sensor_inference/cfgs/detection_lidar_large.yaml) with the origin pointpillar using the 20% training data and test on validation set of WOD, the mAP of LSD is:
 Vec_L1      Vec_L2        Ped_L1       Ped_L2      Cyc_L1  
   Cyc_L2
75.06/74.44    66.34/65.77    72.10/61.43    63.75/54.12    64.55/63.03    62.35/60.88

**Comment by w111liang222** [comment]
Hi, 
We use the OpenPCDet framework to train the models in LSD, it supports to use custom dataset.

We compared the LSD's lidar detection model(sensor_inference/cfgs/detection_lidar_large.yaml) with the origin pointpillar using the 20% training data and test on validation set of WOD, the mAP of LSD is:
 Vec_L1      Vec_L2        Ped_L1       Ped_L2      Cyc_L1  
   Cyc_L2
75.06/74.44    66.34/65.77    72.10/61.43    63.75/54.12    64.55/63.03    62.35/60.88
